packages/web-config-server/src/apiV1/measureBuilders/valueForOrgGroup.js

```diff
@@ -55,11 +55,19 @@ class ValueForOrgGroupMeasureBuilder extends DataBuilder {
       organisationUnitCode: this.entity.code,
     });
 
-    const analytics = results.map(result => ({
+    let analytics = results.map(result => ({
       ...result,
       value: result.value === undefined ? '' : result.value.toString(),
     }));
 
+    // Temp solution for RN-523
```

Comment:
We decided to go for a temp solution because of time pressure + we would be able to do this easily in non-legacy visuals, and this is a throwaway legacy viz.

Reply:
Do we have a ticket to switch this viz to a non-legacy and remove this code? That should be a fairly simple task yeah?

Reply:
No ticket but the project is still in early stages, we need to migrate the data sources to the new data service and then after we can switch the vizes